ProcessLib/CoupledSolutionsForStaggeredScheme.h

```diff
@@ -76,9 +76,13 @@ struct LocalCoupledSolutions
 
 std::vector<std::vector<double>> getPreviousLocalSolutions(
     const CoupledSolutionsForStaggeredScheme& cpl_xs,
-    const std::vector<GlobalIndexType>& indices);
+    const std::vector<
+        std::reference_wrapper<const std::vector<GlobalIndexType>>>&
+        indices);
 
 std::vector<std::vector<double>> getCurrentLocalSolutions(
```

Comment:
Just for better understanding: why is Solution**s** used in plural form? Could this be somehow documented?

I understand that this is not introduced in this PR.

Reply:
Documented these two functions.